kotlin_client/src/main/java/com/github/paylike/kotlin_client/request/PaylikeRequestBuilder.kt

```diff
@@ -0,0 +1,39 @@
+package com.github.paylike.kotlin_client.request
+
+import java.util.concurrent.Future
+
+// PaylikeRequestBuilder provides a flexible way to add
```

Comment:
Are these comments picked up by the Intellisense? 

I've been using [KDoc](https://kotlinlang.org/docs/kotlin-doc.html#kdoc-syntax) assuming that's the only one getting picked up 😄